codec_packetizers/h264/h264_depacketizer.c

```diff
@@ -146,10 +170,14 @@ static H264Result_t DepacketizeAggregationPacket( H264DepacketizerContext_t * pC
             pCtx->curPacketIndex += naluLength;
         }
     }
+    else
+    {
+        result = H264_RESULT_MALFORMED_PACKET;
+    }
 
     /* If we do not have enough data left in this packet, move to the next
      * packet in the next call to H264Depacketizer_GetNalu. */
```

Comment:
```suggestion
    /* Move to the next packet once current packet content is fully parsed. */
```

By changing this, we should update the comment for it.

Reply:
Done.